README.md

```diff
@@ -1,340 +1,535 @@
+"ssv-dkg:latest" /app init --configPath /data/initiator.yaml && \
+docker rm ssv_dkg_initiator
 ```
 
-### Initiator
+Just make sure to substitute `<PATH_TO_FOLDER_WITH_CONFIG_FILES>` with the actual folder containing all the files.
+You can, of course, change the configuration above to one that suits you better, just be mindful about changing the path references in the docker command **and** in the `operator.yaml` file as well.
+
+#### Build from source
 
-The initiator uses `init` to create the initial details needed to run DKG between all operators.
+To build from source you'll need to have Go version 1.20 installed on your system
 
-Run:
+##### Build
 
+A prerequisite for this is to have `go` version 1.20 installed on the system, and an optional requirement is to have the `make` tool installed as well (alternatively you could run the corresponding command defined in the `Makefile`).
+```sh
+make install
+```
+##### Launch with command line parameters
+
+It is advised to store all the necessary files (operators_info.json, encrypted_private_key.json, password) in a single folder (in this case initiator-config), as shown below:
+```sh
+ssv@localhost:~/ssv-dkg# tree initiator-config
+initiator-config
+├── encrypted_private_key.json
+├── operators_info.json
+└── password
+
+1 directory, 3 files
+```
+
+The Initiator creates the initial details needed to run DKG between all operators via the init command. You can launch the following command with the appropriate values to each parameter:
 ```sh
 ssv-dkg init \
           --operatorIDs 1,2,3,4 \
           --operatorsInfoPath ./examples/operators_integration.json \
           --owner 0x81592c3de184a3e2c0dcb5a261bc107bfa91f494 \
           --nonce 4 \
-          --withdrawAddress 0000000000000000000000000000000000000009  \
-          --network "mainnet" \
-          --outputPath ./output/ \
-          --initiatorPrivKey ./examples/initiator/encrypted_private_key.json \
-          --initiatorPrivKeyPassword ./examples/initiator/password \
+          --withdrawAddress 0xa1a66cc5d309f19fb2fda2b7601b223053d0f7f4  \
+          --fork "mainnet" \
+          --depositResultsPath deposit.json \
+          --ssvPayloadResultsPath payload.json \
+          --initiatorPrivKey ./encrypted_private_key.json \
+          --initiatorPrivKeyPassword ./password \
           --logLevel info \
           --logFormat json \
           --logLevelFormat capitalColor \
-          --logFilePath ./examples/output/initiator_debug.log
-
-#### where
---operatorIDs 1,2,3,4 # operator IDs which will be used for a DKG ceremony
-###### Operators info data part.
-###### operatorsInfoPath or operatorsInfo, not both.
---operatorsInfoPath ./operators_integration.json # path to operators info file or directory.
---operatorsInfo '[{"id": 1,"public_key": "LS0tLS1CRUdJTiBSU0....","ip": "http://localhost:3030"}, {"id": 2,"public_key": "LS0tLS1CRUdJTiBSU0....","ip": "http://localhost:3030"},...]' # raw JSON string containing operators info.
```

Comment:
this is not mentioned anywhere and you deleted it. (the correct format for this flag)

Reply:
I initially only added it to the docker YAML, now it's in the other two options (source, source-with-yaml) as well